backend/inspirehep/assign/views.py

```diff
@@ -166,7 +166,7 @@ def assign_different_profile(args):
 
     for literature_id in literature_ids:
         record = LiteratureRecord.get_record_by_pid_value(literature_id)
-        if record.get("curated") and not is_from_author_stub:
+        if record.get("curated_relation") and not is_from_author_stub:
```

Comment:
this is incorrect, you need to check the `curated_relation` field from the matched author, there is no `curated_relation` in the record. It might be good to add a test too for this.

Reply:
Ok, I see thanks. I misunderstood your requirement